src/executors/node.yml

```diff
@@ -0,0 +1,9 @@
+description: >
+  Default executor for node.js jobs
+docker:
+  - image: 'circleci/node:<<parameters.tag>>'
+working_directory: ~/app
+parameters:
+  tag:
+    default: "10-buster-browsers"
```

Comment:
don't think you need `buster` - won't that just pin on buster rather than the latest?

Reply:
I took this from one of the projects - I think we went with buster for the Dockerfile (because it had less vulnerabilities), so we used the same for the cirleci jobs.

Reply:
yes, but my point is that the default is buster anyway